--- a/prog/sensord/rrd.c
+++ b/prog/sensord/rrd.c
@@ -204,6 +204,10 @@
         min = "-100";
         max = "250";
         break;
+      case DataType_loadavg:
+        min = "0";
+        max = "U";
+        break;
       default:
         min = max = "U";
         break;
@@ -219,8 +223,11 @@
   int ret = 0;
   struct ds data = { 0, argv};
   ret = applyToFeatures (rrdGetSensors_DS, &data);
-  if (!ret && doLoad)
-    ret = rrdGetSensors_DS (&data, LOADAVG, LOAD_AVERAGE, NULL);
+  if (!ret && doLoad) {
+    FeatureDescriptor feature;
+    feature.type = DataType_loadavg;
+    ret = rrdGetSensors_DS (&data, LOADAVG, LOAD_AVERAGE, &feature);
+  }
   return ret ? -1 : data.num;
 }
 
--- a/prog/sensord/sensord.h
+++ b/prog/sensord/sensord.h
@@ -80,6 +80,7 @@
   DataType_voltage = 0,
   DataType_rpm,
   DataType_temperature,
+  DataType_loadavg,
   DataType_other = -1
 } DataType;
 
